Logger/Student.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace Logger;
+
+public record Student : IEntity
+{
+    private FullName _name;
+    public int Sid { get; init; }
```

Comment:
```suggestion
    public int StudentId { get; init; }
```
This is more clear than `Sid`. We don't have to worry about performance or something when it's shorter, so we should opt for readability. If I'm a calling user, `Sid` could be confusing